gammapy/datasets/map.py

```diff
@@ -337,34 +337,53 @@ def data_shape(self):
         """Shape of the counts or background data (tuple)"""
         return self._geom.data_shape
 
-    def _energy_range(self, mask = None):
-        """Compute the energy range with or without the mask fit."""
-        energy = self._geom.axes["energy"].edges
-        energy_min, energy_max = energy[:-1], energy[1:]
+    def _energy_range(self, mask=None):
+        """Compute the energy range maps with or without the fit mask."""
+        geom = self._geom
+        energy = geom.axes["energy"].edges
```

Comment:
have we decided finally to always use the convention e_reco and e_true? If yes, here, it should be e_reco....

Reply:
Its correct we now use consistently `energy` and `energy_true`, see https://docs.gammapy.org/0.18.2/irf/index.html#irf-axis-naming.

Reply:
Thanks Axel for the reminder!